charts/longhorn/100.0.0+up1.1.2/Chart.yaml

```diff
@@ -2,10 +2,11 @@ annotations:
   catalog.cattle.io/auto-install: longhorn-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Longhorn
+  catalog.cattle.io/kube-version: '>=v1.16.0-r0'
```

Comment:
One quick question here, do we need to field `kubeVersion` in this file?

Reply:
AFAIK, no we don't need to add kubeVersion if it does not have one. The kube-version annotation should be sufficient.